src/Sulu/Bundle/AdminBundle/Resources/js/components/ResourceLocator/ResourceLocator.js

```diff
@@ -2,18 +2,37 @@
 import React from 'react';
 import {action, computed, observable} from 'mobx';
 import {observer} from 'mobx-react';
+import type {IObservableValue} from 'mobx';
 import Input from '../Input';
 import resourceLocatorStyles from './resourceLocator.scss';
 
 type Props = {|
     disabled: boolean,
     id?: string,
+    locale: IObservableValue<string>,
     mode: string,
     onBlur?: () => void,
     onChange: (value: ?string) => void,
     value: ?string,
 |};
 
+const replacerMap = new Map([
+    // replace multiple dash with one
+    [/([-]+)/g, '-'],
```

Comment:
I think the order here is important. That should probably be one of the last entries here. Because if I am copying now a text like "That's a /// test!" in the resource locator field, it will still result in multiple dashes.